spaces/tokenlon/index.json

```
@@ -0,0 +1,28 @@
+{
+  "token": "0xE534619dEFDBF0cAf673b8AbF7158714F5BD4bd9",
+  "name": "Tokenlon",
+  "network": "1",
+  "symbol": "TFT",
+  "skin": "tokenlon",
+  "strategies": [
+    {
+      "name": "tokenlon",
```

This should be changed
```suggestion
      "name": "erc20-with-balance",
```